src/index.js

```diff
@@ -0,0 +1,111 @@
+import React, { useRef, useState, useLayoutEffect } from 'react';
+import { makeDecorator } from '@storybook/addons';
+import { addons } from '@storybook/addons';
+import { Resizable } from 're-resizable';
+import constants from './constants';
+
+const {
+  minWrapperWidth,
+  resizeHandleWidth,
+  wrapperWidthChangeEvent,
+  wrapperMaxWidthChangeEvent,
+  toolWidthChangeEvent
+} = constants;
+
+const TaffyWrapper = ({ children }) => {
+
+  let isMounted = false;
+  const channel = addons.getChannel();
+  const wrapperEl = useRef(null);
+  const [wrapperWidth, setWrapperWidth] = useState('100%');
+  useLayoutEffect(() => {
+    isMounted = true;
+    if (wrapperEl) {
+      const maxWidth = wrapperEl.current.getBoundingClientRect().toJSON().width;
```

Comment:
[question] What's the `toJSON()` for? Does `getBoundingClientRect` not return a `DOMRect` object?

Reply:
Whoops -- I have no idea why I wrote that.